devices/common_patches/g4_adc_common.yaml

```diff
@@ -0,0 +1,11 @@
+ADC*_*:
+  CCR:
+    _add:
+      VBATEN:
```

Comment:
Should this be `VBATSEL`? Also it looks like in at least some G4 SVDs (eg G431) these fields already exist with names `CH17SEL` and `CH18SEL`.